features/inventory/validating_product_quantity_on_checkout_complete.feature

```diff
@@ -0,0 +1,43 @@
+@inventory
+Feature: Being unable to buy products that are out of stock
+    In order to be sure that products I buy are available
```

Comment:
Either `... the product i buy...` 
or `In order to buy only available products`